quickstarts/README.md

```diff
@@ -0,0 +1,9 @@
+# Anthos Service Mesh Quickstarts
+
+This folder contains the quickstart material for creating a GKE cluster with Google's Managed Anthos Service Mesh.
+The following are methods showcased:
+* Terraform
```

Comment:
**Question:** Other than `terraform-asm-mcp/`, do we expect the `quickstarts/` folder to contain more folders in the future? Would it be overkill to have a separate `quickstarts/` folder?

Reply:
I was thinking about this. I think having this folder can also lend itself to showing how to do a multi cluster quickstart w/ Terraform, or fleet api quickstart (either single cluster/multicluster). 

This quickstart just deploys the cluster and provisions ASM, and is not tied down to any sample app - I would ideally like to keep sample app setup vs day 0 provisioning separate. I think potentially the name `quickstarts` might be misleading, open to suggestions!